src/app/workspace/results/results.component.ts

```diff
@@ -63,7 +63,7 @@ export class ResultsComponent {
 
     onSelectionChange(res: FilteredResources) {
         this.selectedResources = res;
-        this.resourceIri = this.selectedResources.resInfo[0].id;
+        this.resourceIri = this.selectedResources.resInfo[0] ? this.selectedResources.resInfo[0].id : undefined;
```

Comment:
```suggestion
        this.resourceIri = this.selectedResources.resInfo[0]?.id;
```

Isn't it enough?

Reply:
you are correct, changed in [b53ec4f](https://github.com/dasch-swiss/dsp-app/pull/817/commits/b53ec4f5fd86455eca8b0a04ba5831eb0b071ea7)